test/framework/framework.go

```diff
@@ -97,9 +97,6 @@ func New(kubeconfig, opImage, exampleDir, resourcesDir string, operatorVersion s
 	}
 
 	httpc := cli.CoreV1().RESTClient().(*rest.RESTClient).Client
-	if err != nil {
-		return nil, fmt.Errorf("creating http-client failed: %w", err)
-	}
```

Comment:
this error validation here doesn't make sense. For this reason I'm removing it

Reply:
Ideally, we keep the scope of PRs consistent. I.e. this change would make more sense in another PR. This way if we need to revert one thing, we don't revert unrelated changes.

Since this is suuuuper small, let's just call it nitpicking and approve it :P

Reply:
Ok, If I found something like this one, I will address in a different PR. Thanks for the hint.